packages/@okta/i18n/src/properties/login_fi.properties

```diff
@@ -533,7 +533,7 @@ primaryauth.submit = Kirjaudu sis\u00E4\u00E4n
 primaryauth.newUser.tooltip=T\u00E4m\u00E4 on ensimm\u00E4inen {0} -yhteydenottosi t\u00E4lt\u00E4 selaimelta
 primaryauth.newUser.tooltip.close=Sulje
 oktaVerify.description = Organisaatiosi vaatii sinua kirjautumaan sis\u00E4\u00E4n laitteellasi, ett\u00E4 voit k\u00E4ytt\u00E4\u00E4 t\u00E4t\u00E4 resurssia.
-oktaVerify.button = Kirjaudu sis\u00E4\u00E4n Okta Verifyll\u00E4 t\u00E4ll\u00E4 laitteella
+oktaVerify.button = Kirjaudu sis\u00E4\u00E4n Okta FastPassilla
```

Comment:
Double checking - isn't this a reserved product name? `Fastpass`

Reply:
This is the Finnish language handling of proper nouns. It's part of the grammatical construct.